### PR TITLE
[pino] fix types for symbols (unique symbol)

### PR DESCRIPTION
**Pull request by annahassel**
Fix for error "**A computed property name in a class property declaration must refer to an expression whose type is a literal type or a 'unique symbol' type**".
More about problem [https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html).

### CODE REVIEW

**Comment by sorokinsh** [comment]
@AdamVig Can you help to merge it?

File: types/pino/index.d.ts [hunk 1]
```diff
@@ -47,32 +47,32 @@ declare namespace P {
...
+        readonly endSym: unique symbol,
+        readonly formatOptsSym: unique symbol,
+        readonly messageKeySym: unique symbol,
+        readonly nestedKeySym: unique symbol,
+        readonly wildcardFirstSym: unique symbol,
+        readonly needsMetadataGsym: unique symbol,
+        readonly useOnlyCustomLevelsSym: unique symbol,
+        readonly formattersSym: unique symbol,
+        readonly hooksSym: unique symbol,
     };
```
Comment (AdamVig) [inline types/pino/index.d.ts:76]: I wonder if we would benefit from adding `as const` here.

**Review by AdamVig — APPROVED** [review]
Thanks for your patience, I am not sure how I missed this. Your solution looks good.

edit: You may want to add yourself as a maintainer in the type definitions so that you can help with reviews in the future.

**Comment by sorokinsh** [comment]
Ready to merge

**Comment by sorokinsh** [comment]
@AdamVig Thank you for your help!

**Pull request merged**